cfme/tests/automate/test_domain_priority.py

```diff
@@ -38,6 +38,14 @@ def copy_domain(request):
     return domain
 
 
+@pytest.fixture(scope="module")
+def original_domain(request):
+    domain = Domain(name=fauxfactory.gen_alphanumeric(), enabled=True)
+    domain.create()
+    request.addfinalizer(lambda: domain.delete() if domain.exists() else None)
```

Comment:
It could indeed.....was just following the known to work pattern of the others for now